k8s/configmap.yaml

```diff
@@ -2,18 +2,25 @@ apiVersion: v1
 kind: ConfigMap
 data:
   SCALYR_K8S_CLUSTER_NAME: <your-cluster-name>
-  SCALYR_K8S_DISABLE_API_SERVER: "true"
-  SCALYR_K8S_ALWAYS_USE_DOCKER: "true"
-  SCALYR_K8S_CACHE_EXPIRY_SECS: "600"
-  SCALYR_K8S_CACHE_START_FUZZ_SECS: "10"
-  SCALYR_K8S_CACHE_EXPIRY_FUZZ_SECS: "15"
-  SCALYR_COMPRESSION_TYPE: bz2
-  K8S_EVENTS_DISABLE: "true"
+  SCALYR_K8S_RATELIMIT_CLUSTER_NUM_AGENTS: "1000"
+  SCALYR_K8S_RATELIMIT_CLUSTER_RPS_INIT: "40"
+  SCALYR_K8S_RATELIMIT_CLUSTER_RPS_MIN: "8"
+  SCALYR_K8S_RATELIMIT_CLUSTER_RPS_MAX: "200"
+  SCALYR_K8S_RATELIMIT_CONSECUTIVE_INCREASE_THRESHOLD: "5"
+  SCALYR_K8S_RATELIMIT_STRATEGY: "multiply"
+  SCALYR_K8S_RATELIMIT_INCREASE_FACTOR: "2.0"
+  SCALYR_K8S_RATELIMIT_BACKOFF_FACTOR: "0.5"
+  SCALYR_K8S_RATELIMIT_MAX_CONCURRENCY: "1"
+  SCALYR_K8S_USE_CONTROLLED_WARMER: "true"
+  SCALYR_K8S_EVENTS_DISABLE: "true"
   SCALYR_REPORT_CONTAINER_METRICS: "false"
   SCALYR_REPORT_K8S_METRICS: "false"
-  SCALYR_ENABLE_PROFILING: "true"
-  SCALYR_PROFILE_DURATION_MINUTES: "2"
-  SCALYR_MAX_PROFILE_INTERVAL_MINUTES: "30"
+  SCALYR_COMPRESSION_TYPE: bz2
+  SCALYR_MAX_LINE_SIZE: "49900"
+  SCALYR_READ_PAGE_SIZE: "131072"
```

Comment:
Not convinced this needs to be set.  The default is 64k, and we don't read more than the line size anyway.  The max_line_size isn't a hard limit (it might occasionally go over this), so leaving the default is ok.  I don't imagine they'll want to change it.